controls/radwindow/overview.md

```diff
@@ -12,42 +12,29 @@ previous_url: http://www.telerik.com/help/wpf/radwindow-overview.html
 
-![Rad Window Overview 03](images/RadWindow_Overview_03.png)
+![Rad Window Overview 03](images/radwindow-overview.png)
 
-This is a list with short descriptions of the top-of-the-line features of Telerik's __RadWindow__ control:
+This is a list with short descriptions of the top-of-the-line features of Telerik's __RadWindow__ control.
 
-* __Child Windows Across All Platforms and Scenarios__ - Telerik __RadWindow__ makes possible to create Child Windows in Silverlight, WPF and XBAP applications with the same API, functionality and appearance across all platforms and scenarios.
+* __Child Windows Across All Platforms__: Telerik __RadWindow__ enables you to create child windows in Silverlight, WPF and XBAP applications with the same API, functionality and appearance across all platforms. The [RadWindowManager]({%slug radwindow-features-radiwindowmanager%}) class makes it easy to control all currently open RadWindows.
 
-* __Predefined Dialogs__ - Telerik __RadWindow__ provides an easy way to replace the standard non-customizable dialog boxes (alert, confirm, and prompt) provided by the browser. With __RadWindow__ you can blend the popup into the application’s design to achieve better user experience. [Read more]({%slug radwindow-features-predefined-dialogs%})
+* __Predefined Dialogs__: Telerik __RadWindow__ provides an easy way to replace the standard non-customizable dialog boxes (alert, confirm, and prompt) provided by the browser. With __RadWindow__ you can blend the popup into the application’s design to achieve better user experience. [Read more]({%slug radwindow-features-predefined-dialogs%})
 
-* __Styling and Appearance__ - Telerik __RadWindow__ is a fully customizable control, which lets you easily modify all its visual elements, such as borders, resize grips, titlebars, buttons, etc. The control comes with several built-in skins that can be customized if needed. [Read more]({%slug radwindow-styles-and-templates-overview%})
+* __Styling and Appearance__: Telerik __RadWindow__ is a fully customizable control, which lets you easily modify all its visual elements. The control comes with several built-in skins that can be customized if needed. [Read more]({%slug radwindow-styles-and-templates-overview%})
 
-* __Customizable Behavior__ - Telerik __RadWindow__ basic behavior can be configured with a few clicks. You have a full control over determining the initial [state]({%slug radwindow-features-states%}) of the window and whether it can be [resized]({%slug radwindow-features-resizing%}).
+* __Configuration__: Telerik __RadWindow__ can be easily configured to suit the needs of your application. You have a full control over determining the initial [state]({%slug radwindow-features-states%}) and [position]({%slug radwindow-features-positioning%}) of the window, whether it can be [resized]({%slug radwindow-features-resizing%}), [closed]({%slug radwindow-features-closing%}) or [moved]({%slug radwindow-features-moving%}).
 

```

Comment:
Can we add a note pointing to the Getting Started article of the control? Check the [RadTabControl Overview](http://docs.telerik.com/devtools/wpf/controls/radtabcontrol/overview2) article.